#! /usr/bin/env python

PACKAGE='basic_dynamic_reconfigure_tutorial'
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name              Type          level               Description              Default    Min         Max
gen.add("param_int",      int_t,        0,                  "Parameta Int",          1,         -9,         9)
gen.add("param_double",   double_t,     0,                  "Parameta Dobule",       1.0,        -9.9,       9.9)
gen.add("param_string",   str_t,        0,                  "Parameta String",       "Pigeon")
gen.add("param_bool",     bool_t,       0,                  "Parameta Bool",         True)

exit(gen.generate(PACKAGE, "dynamic_reconfigure_tutorial", "Tutorial"))

